external-crates/move/crates/move-compiler/tests/development/enums/expansion/constants_dont_shadow_leading_access_struct.exp

```diff
@@ -1,9 +1,9 @@
 error[E03006]: unexpected name in this position
    ┌─ tests/development/enums/expansion/constants_dont_shadow_leading_access_struct.move:15:13
    │
+ 3 │     public struct S() has copy, drop;
+   │                   - But 'S' is an struct
+   ·
 15 │             S::foo(); // resolves to struct
-   │             ^
-   │             │
-   │             Invalid construction. Expected an enum
-   │             But 'S' is a struct
+   │             ^ Invalid constructor. Expected an enum
```

Comment:
Thoughts on the name switch from "construction" to "constructor"?
I have no strong opinions here but I recall trying to pick a name here and struggling. I wanted to:
- make sure folks knew what the fuck I was talking about :P hence its not called "Pack"
- avoid being too misleading, particularly I was worried that folks would think that code was being invoked when the struct was packed (instead of just a simple data transformation)

Thoughts given this?

Reply:
Given (b), I think `construction` is clearer, as `constructor` is a _very_ overloaded term (cf. C++).